pkg/issuer/acme/http/ingress.go

```diff
@@ -137,9 +145,8 @@ func buildIngressResource(ch *cmacme.Challenge, svcName string) (*extv1beta1.Ing
 	}
 
 	podLabels := podLabels(ch)
-	// TODO: add additional annotations to help workaround problematic ingress controller behaviours
+
 	ingAnnotations := make(map[string]string)
-	ingAnnotations["nginx.ingress.kubernetes.io/whitelist-source-range"] = "0.0.0.0/0,::/0"
```

Comment:
We can't just remove this annotation now as some users will have come to depend on it. Removing it may be possible but will require some careful thinking around API defaulting, which for the time being I'd rather avoid.

Let's just keep this as a default annotation for now - it's not ideal, but if you then open an issue to track us removing this default behaviour, we can investigate how to do it properly without hindering this PR 😄

Reply:
Makes sense. I've added a comment and the annotation back in.